src/util/mt_inotify.cc

```diff
@@ -55,12 +55,14 @@
 #define MAX_EVENTS 4096
 #define MAX_STRLEN 4096
 
-Inotify::Inotify()
 #ifdef __linux__
-    : inotify_fd(inotify_init1(IN_CLOEXEC))
+static constexpr bool linux = true;
 #else
-    : inotify_fd(inotify_init())
+static constexpr bool linux = false;
 #endif
+
+Inotify::Inotify()
+    : inotify_fd(linux ? inotify_init1(IN_CLOEXEC) : inotify_init())
```

Comment:
I'm not sure whether this works on all systems. The symbol may be undefined.

Reply:
IN_CLOEXEC?

Reply:
`inotify_init` or `inotify_init1`

Reply:
CI seems to have no issue.